lib/components/inputs/inputview.coffee

```diff
@@ -235,15 +235,22 @@ module.exports = class KDInputView extends KDView
   ###*
    * Set the value of this input field.
   ###
-  setValue:(value)->
-    $el = @$()
-    el  = $el[0]
-    if @getOption("type") in ["checkbox", "radio"]
+  setValue: (value) ->
+
+    $el    = @$()
+    [ el ] = $el
+
+    if @getOption('type') in [ 'checkbox', 'radio' ]
       if value
-      then el.setAttribute "checked", "checked"
-      else el.removeAttribute "checked"
+        el.setAttribute 'checked', 'checked'
+        el.checked = yes
+      else
+        el.removeAttribute 'checked'
+        el.checked = no
+
```

Comment:
I spend half an hour see what's going on here and fix it. The problem was if the input is checked by clicking the DOM element then `setValue no` wasn't working.